### PR TITLE
Added docblocks

### PR DESCRIPTION
**Pull request by Tjoosten**
referrer to #324

### CODE REVIEW

File: Parsedown.php [hunk 1]
```diff
@@ -55,8 +78,12 @@ function setBreaksEnabled($breaksEnabled)
         return $this;
     }
 
-    protected $breaksEnabled;
-
+    /**
+     * setMarkupEscaped()
```
Comment (GrahamCampbell) [inline Parsedown.php]: :-1: for all these descriptions

**Comment by GrahamCampbell** [comment]
:-1:

**Comment by GrahamCampbell** [comment]
Most of these phpdocs are wrong or pointless.

**Comment by Tjoosten** [comment]
Ok sorry u was trying to help

File: Parsedown.php [hunk 2]
```diff
@@ -283,6 +324,13 @@ private function lines(array $lines)
     #
     # Code
 
+    /**
+     * blockCode()
+     *
+     * @param $Line
```
Comment (GrahamCampbell) [inline Parsedown.php]: this is no s unless you include the type

File: Parsedown.php [hunk 3]
```diff
@@ -283,6 +324,13 @@ private function lines(array $lines)
...
+     * @param null $Block
```
Comment (GrahamCampbell) [inline Parsedown.php]: this type is wrong

File: Parsedown.php [hunk 4]
```diff
@@ -309,6 +357,13 @@ protected function blockCode($Line, $Block = null)
         }
     }
 
+    /**
+     * blockCodeContinu()
+     *
+     * @param $Line
```
Comment (GrahamCampbell) [inline Parsedown.php]: and it's the same for all of it...

**Comment by cebe** [comment]
there is no sense in adding docblocks that could be autogenerated by an IDE as they do not add any value to what is already visible from the code. If adding phpdoc comments they should contain useful description.

**Comment by nblackburn** [comment]
Adding docblocks is a good idea but they at least need to be descriptive.

**Comment by GrahamCampbell** [comment]
> Adding docblocks is a good idea but they at least need to be descriptive.

And actually have correct syntax. Nearly every param annotation here is missing the type so is incorrect syntax. In some cases the types are even wrong too.

**Pull request closed without merging**